cdp_scrapers/instances/houston.py

```diff
@@ -0,0 +1,283 @@
+import logging
+from datetime import datetime
+from typing import Any, List, Optional, Union
+
+import requests
+from bs4 import BeautifulSoup, NavigableString, Tag
+from cdp_backend.pipeline import ingestion_models
+from cdp_backend.pipeline.ingestion_models import EventIngestionModel
+
+from cdp_scrapers.scraper_utils import IngestionModelScraper
+
+log = logging.getLogger(__name__)
+
+
+class HoustonScraper(IngestionModelScraper):
+    def __init__(self):
+        super().__init__(timezone="America/Chicago")
+
+    def remove_extra_type(self, element: Union[Tag, NavigableString, None]) -> Tag:
+        """
+        Remove types that are not useful
+
+        Parameter:
+        ----------------
+        event: Union[Tag, NavigableString, None]
+            All elements in the page that we want to scrape
```

Comment:
I don't really agree with this docstring. The word `element` implies a single object, not "all elements...". This just looks like some copy-paste job that was forgotten to be fixed later. There are other docstrings below with the same issue.

Reply:
Done